conf/solrconfig.xml

```diff
@@ -1334,6 +1334,37 @@
     </arr>
   </requestHandler>
 
+  <searchComponent class="solr.SuggestComponent" name="suggest">
+    <lst name="suggester">
+      <str name="name">analyzingInfixSuggester</str>
+      <str name="lookupImpl">FuzzyLookupFactory</str>
+      <str name="dictionaryImpl">DocumentDictionaryFactory</str>
+      <str name="field">fgs_label_t</str>
+      <str name="weightField">suggest_weight</str>
+      <str name="suggestAnalyzerFieldType">text</str>
+      <str name="buildOnCommit">true</str>
+    </lst>
+    <lst name="suggestertwo">
```

Comment:
I don't believe we need two suggesters. Seems to work fine without the second one.

Reply:
Whoopsie, removed the second suggester.